### ISSUE TITLE
[issue]: Macrium Reflect 8.0.6392 ISO WinRE 64-bit doesn't boot (put as ISO)

### ISSUE THREAD

**Issue by rwasef1830** [ventoy/Ventoy#1397]
### Official FAQ

- [X] I have checked the official FAQ.

### Ventoy Version

1.0.64

### What about latest release

Yes. I have tried the latest release, but the bug still exist.

### BIOS Mode

UEFI Mode + Secure Boot (Key already enrolled by MokManager)

### Partition Style

MBR

### Disk Capacity

1TB

### Disk Manufacturer

PNY

### What happened?

This used to work with older Ventoy release, but I tested a brand new created Macrium Reflect 8.0 Workstation Rescue ISO.
Ventoy shows a black screen with cursor. After that it returns again to the ventoy menu.
I'm booting from laptop built-in SD card reader Dell Precision 7510.

**Comment by rwasef1830** [comment]
More information:
I put the same non-booting iso on a USB with an older version of Ventoy, it also does the same behavior.
I found an older working reflect macrium rescue ISO, it was made from build 8.0.5928. But an ISO made from the latest released version available now does not boot.
I put the older working macrium ISO, on the memory card with the new ventoy and booted from the laptop built-in reader. It booted.

Conclusion: Something was changed by Macrium in newer 8.0 builds which make it not compatible with Ventoy.
I will test deleting the Macrium boot files cache and rebuilding the ISO from scratch.

**Comment by rwasef1830** [comment]
More info, I checked the files in the root. All of them are identical (bootmgfw.efi and bootmgr.efi) and rest of XML and text files containing rescue media version and enabled features. The only difference is bootmgr file. I attached zip file containing working and not working bootmgr from both ISOs.

[bootmgr-working-and-not-working.zip](https://github.com/ventoy/Ventoy/files/7899240/bootmgr-working-and-not-working.zip)

**Comment by rwasef1830** [comment]
Rebuild of rescue media after deleting all the cached files didn't help. The new ISO still doesn't boot.
I attached BCD from both. Let me know if you need any files from both to compare.
[BCD-working-and-notworking.zip](https://github.com/ventoy/Ventoy/files/7899437/BCD-working-and-notworking.zip)

**Comment by rwasef1830** [comment]
Comparing the 2 BCDs:

Working:
----------
```
Windows Boot Manager
--------------------
identifier              {9dea862c-5cdd-4e70-acc1-f32b344d4795}
description             Windows Boot Manager
locale                  en-US
inherit                 {7ea2e1ac-2e61-4728-aaa3-896d9d0a9f0e}
default                 {7619dcc9-fafe-11d9-b411-000476eba25f}
displayorder            {7619dcc9-fafe-11d9-b411-000476eba25f}
toolsdisplayorder       {b2721d73-1db4-4c62-bf78-c548a880142d}
timeout                 30

Windows Boot Loader
-------------------
identifier              {7619dcc9-fafe-11d9-b411-000476eba25f}
device                  ramdisk=[boot]\sources\boot.wim,{7619dcc8-fafe-11d9-b411-000476eba25f}
path                    \windows\system32\boot\winload.efi
description             Windows Setup
locale                  en-US
inherit                 {6efb52bf-1766-41db-a6b3-0ee5eff72bd7}
isolatedcontext         Yes
osdevice                ramdisk=[boot]\sources\boot.wim,{7619dcc8-fafe-11d9-b411-000476eba25f}
systemroot              \windows
bootmenupolicy          Standard
detecthal               Yes
winpe                   Yes
ems                     No
```

Not working:
```
Windows Boot Manager
--------------------
identifier              {9dea862c-5cdd-4e70-acc1-f32b344d4795}
description             Windows Boot Manager
locale                  en-US
inherit                 {7ea2e1ac-2e61-4728-aaa3-896d9d0a9f0e}
default                 {7619dcc9-fafe-11d9-b411-000476eba25f}
displayorder            {7619dcc9-fafe-11d9-b411-000476eba25f}
toolsdisplayorder       {b2721d73-1db4-4c62-bf78-c548a880142d}
timeout                 30

Windows Boot Loader
-------------------
identifier              {7619dcc9-fafe-11d9-b411-000476eba25f}
device                  ramdisk=[boot]\sources\boot.wim,{7619dcc8-fafe-11d9-b411-000476eba25f}
path                    \windows\system32\boot\winload.exe
description             Windows Setup
locale                  en-US
inherit                 {6efb52bf-1766-41db-a6b3-0ee5eff72bd7}
osdevice                ramdisk=[boot]\sources\boot.wim,{7619dcc8-fafe-11d9-b411-000476eba25f}
systemroot              \windows
bootmenupolicy          Standard
detecthal               Yes
winpe                   Yes
ems                     No
```

winload.exe vs winload.efi
The boot.wim contains both files in this path. Is this a bug in Macrium or both should work ?

**Comment by rwasef1830** [comment]
WIMBoot also doesn't work

**Comment by ventoy** [comment]
Please share me the ISO file for test.

**Comment by rwasef1830** [comment]
@ventoy is there a private place to share the file with you, it's made from paid version of macrium and i can't share it publicly.

**Comment by rwasef1830** [comment]
@ventoy kindly you can find my wetransfer mail containing the non-working ISO.
I hope it makes the problem easy to find. Thank you so much.

**Comment by ventoy** [comment]
I test the iso file in VMWare in my laptop.  In legacy BIOS mode it boot OK.
But it can not boot in UEFI mode from CD/DVD, so the iso file itself must have some problem.
Ventoy emulate the ISO file as a CDROM when boot it. So if the ISO file can not boot from CD/DVD then Ventoy can not boot it.

**Comment by rwasef1830** [comment]
@ventoy do you see anything wrong with the ISO why it may not be booting ? is there anything wrong in UEFI structure ?

**Comment by rwasef1830** [comment]
@ventoy I want to see this file, how to access this hidden filesystem ? Is there any tool that can get this file ? I want to compare its hash with files in my WinRE partition to find out where it's coming from. 7-zip only shows the UDF file system.

**Comment by rwasef1830** [comment]
I have extracted the bootx64.efi from boot image vs from UDF file system from both working and not working ISO for testing and comparison. I hope this helps.

[Troubleshoot-Macrium.zip](https://github.com/ventoy/Ventoy/files/7919474/Troubleshoot-Macrium.zip)

**Comment by rwasef1830** [comment]
Macrium support has told me a fix for this issue is coming in the next release. Will close this issue when confirmed working.

**Comment by Fasto5** [comment]
Thank you for posting this detailed thread and for continuing to update it - I am having the same issue (with the .ISO created by Macrium v8.0.6392).

**Comment by rwasef1830** [comment]
@Fasto5 @ventoy issue solved by update to Macrium 8.0.6560 and rebuilding the rescue media.
